app/javascript/controllers/projects/samples/attachments/selected_attachments_controller.js

```diff
@@ -18,7 +18,13 @@ export default class extends Controller {
 
     if (storageValues) {
       for (let [storageValueIndex, storageValue] of storageValues.entries()) {
-        const value = JSON.parse(storageValue);
+        let value = ''
```

Comment:
Because of the uuids, I had to `stringify` the `storageValue` for proper behaviour. However, paired attachment work fine with the `stringify`, and if you were to `stringify` it, the array brackets become an issue.

Reply:
Placed in try catch now c4fd0f82335962b11b2bca604631bd5ce0389dd8